script/Ciel/Day 7/07_01_CIEL07_2_1MERGE.txt

```diff
@@ -380,93 +395,95 @@ Senpai even offers me her sympathies.
 [sha:1ce451ffcffce217698476b4be0a1f71ca1db7b8]{
 -- Page 219, Offset 15321.
 -- ……確かに、背中が寒くなるような状況は何度かあったけど……。
-True enough, there were countless situations where I felt the chills run through my spine, but...
+I mean, sure, I ended up in plenty of situations that sent chills down my spine, but...
 }
 [sha:7bfe12967b99bb17567d822daeafd46bdd58dc5a]{
 -- Page 220, Offset 15322.
 -- 「……でも、それも終わった話です。ヴローヴを倒した以上、アイツとはもう関係がないんですから」
-"But that's already settled. After all, with Vlov defeated, I have nothing to do with her anymore."
+"But that's all in the past now.
+ Now that Vlov's been defeated, I don't have any reason to see her again."
 }
 [sha:9cbc43457fcc38f9df9c80c644f196c0ebf44b33]{
 -- Page 220, Offset 15323.
 -- ……そうだ。もうアイツの無茶なワガママを耳にする機会は、永遠にやってはこない。
-Right... Never again, will I get the opportunity to hear her absurd demands.
+That's right... I'll never get another chance to listen to her absurd demands.
 }
 [sha:a73686855d4db19ee524a0d42228fc97c7bc6a1e]{
 -- Page 221, Offset 15324.
 -- 「……すみません。もう少し確認しておきますけど。
-"Sorry, but there's still something I have to confirm.
+"Sorry, but there's one more thing I need to confirm...
 }
 [sha:3accab8fc92428c75e3059bb62a6cd7ae4e032fc]{
 -- Page 221, Offset 15325.
 -- 遠野くん、彼女にどれくらいの事を聞いてるんですか？」
-Tohno-kun, how much did you hear from her?"
+Tohno-kun, just how much did she tell you?"
 }
 [sha:81519c39c403783057947558f260528c34d4711a]{
 -- Page 222, Offset 15326.
 -- 「どのくらいの事って……この街に人間を食い物にする吸血鬼がいて、それを追いかけて来たって事ぐらいだけど」
-How much did I hear? Only that there are vampires in this city who prey on humans, and that she came chasing after them.
+"How much did she tell me...?
+ Well, just that there was a vampire in this city that preys on humans, and how she came here chasing after them."
```

Comment:
'after them' doesn't read perfectly, maybe 'after it'? Not to belittle the personhood of vampires of course